.gitignore

```diff
@@ -39,3 +39,4 @@ git-ask-pass.sh
 pprof
 pkg/apiclient/sensor/sensor.swagger.json
 pkg/apiclient/eventsource/eventsource.swagger.json
+/argoexec-linux-amd64
```

Comment:
@alexec Was this committed in error? If so, can I remove?

Reply:
I've a follow-up PR to fix this: https://github.com/argoproj/argo-workflows/pull/5335 - you could approve that?

Reply:
Done